cmd/agent/app/processors/thrift_processor.go

```diff
@@ -106,6 +110,7 @@ func (s *ThriftProcessor) processBuffer() {
 		protocol := s.protocolPool.Get().(thrift.TProtocol)
 		protocol.Transport().Write(readBuf.GetBytes())
 		s.server.DataRecd(readBuf) // acknowledge receipt and release the buffer
+		s.logger.Debug("Span(s) received by the agent")
```

Comment:
might be useful to pull `readBuf.GetBytes()` into a var and include its length in the log statement